__tests__/app.test.js

```diff
@@ -241,8 +241,54 @@ describe("POST: /api/articles/:article_id/comments", () => {
       });
   });
 });
-//username doesnt exist - 404
-//2 for pathway
+
+describe('PATCH: /api/articles/:article_id', ()=>{
+  test('200: should return with a 200 status and sucessfully update the votes on the selected article',()=>{
+    const inc_votes = {inc_votes: 1}
+    return request(app)
+    .patch('/api/articles/3')
+    .send(inc_votes)
+    .expect(200)
+    .then((response) =>{
+      const article = response.body
+      expect(article).toHaveProperty("votes", 1);
+      expect(article).toHaveProperty("title", expect.any(String));
+    })
+  })
+  test('400: should return 400 status code and bad request when inc_votes is a string', ()=>{
```

Comment:
Perhaps a bit of a nitpick, but I'd consider rewording this test description to 'when inc_votes is not a number / the wrong data type' as it's not it being a string specifically that we're interested in, rather that it's not the data type it should be